analytics.js

```diff
@@ -1033,18 +1131,7 @@ class Analytics {
     }
   }
 
-  /**
-   * Call control pane to get client configs
-   *
-   * @param {*} writeKey
-   * @memberof Analytics
-   */
-  load(writeKey, serverUrl, options) {
-    // logger.debug("inside load ");
-    if (this.loaded) return;
-
-    // check if the below features are available in the browser or not
-    // If not present dynamically load from the polyfill cdn
+  isPolyfillPresent() {
```

Comment:
Change the name. May be `arePolyfillsRequired`.

Reply:
Addressed.